III.tdd-katas/fizz-buzz/fizz-buzz.js

```diff
@@ -0,0 +1,20 @@
+
+var newArray = [];
+function fizzbuzz(item){
+    
+if (item % 3 == 0 && item % 5 == 0) {
+    item = "FizzsBuzz";
+        } else 
+        if (item % 3 == 0) {
+    item = "Fizz";
+
+        } else if (item % 5 == 0) {
+    item = "Buzz";
+           
+        }
+    newArray.push(item);
+    for(var i =0; i< newArray.length; i++){
```

Comment:
You're looping through everything in `newArray` here, but not doing anything inside the loop. It can be removed if it does nothing.